ff2mpv.py

```diff
@@ -11,8 +11,9 @@
 def main():
     message = get_message()
     url = message.get("url")
+    time = message.get("time")
 
-    args = ["mpv", "--no-terminal", "--", url]
+    args = ["mpv", "--no-terminal", f"--start={time}", "--", url] if time else ["mpv", "--no-terminal", "--", url]
```

Comment:
Something like this is preferred:

```python
extra_args = []
if time is not None:
    extra_args.append(f"--time={time}")

args = ["mpv", "...", *extra_args, "--", url]
```

Reply:
That gives us more flexibility to add other options in the future, with less diff churn.